src/components/configuration/DataLakeAlertsConfig.vue

```diff
@@ -0,0 +1,622 @@
+const availableVariablesMap = ref<Record<string, DataLakeVariable>>({})
+const isDialogOpen = ref(false)
+const isDeleteDialogOpen = ref(false)
+const isEditing = ref(false)
+const alertToDelete = ref<DataLakeAlert | null>(null)
+
+// Default alert values
+const defaultAlert: Omit<DataLakeAlert, 'id'> = {
+  name: '',
+  enabled: true,
+  conditions: [],
+  level: AlertLevel.Warning,
+  minInterval: 60000,
+  minDuration: 0,
+  customMessage: '',
+}
+
+const editingAlert = ref<
+  Omit<DataLakeAlert, 'id'> & {
+    /**
+     * Optional ID for editing an existing alert
+     */
+    id?: string
+  }
+>({ ...defaultAlert })
+
+// Computed
+const availableVariables = computed(() => {
+  return Object.entries(availableVariablesMap.value).map(([id, variable]) => ({
+    value: id,
+    label: `${variable.name} (${variable.type})`,
+  }))
+})
+
+/**
+ * Get the variable type for a specific condition
+ * @param {AlertCondition} condition - The condition to get the variable type for
+ * @returns {DataLakeVariableType | undefined} The variable type
+ */
+const getConditionVariableType = (condition: AlertCondition): DataLakeVariableType | undefined => {
+  if (!condition.variableId) return undefined
+  return availableVariablesMap.value[condition.variableId]?.type
+}
+
+/**
+ * Get available operators for a specific condition based on its variable type
+ * @param {AlertCondition} condition - The condition to get the operators for
+ * @returns {Array<{value: string, label: string}>} The available operators
+ */
+const getOperatorsForCondition = (
+  condition: AlertCondition
+): Array<{
+  /** The operator value */
+  value: string
+  /** The operator label */
+  label: string
+}> => {
+  const type = getConditionVariableType(condition)
+  if (!type) return []
+  return getComparisonOperatorsForType(type)
+}
+
+const editingAlertIntervalSeconds = computed({
+  get: () => editingAlert.value.minInterval / 1000,
+  set: (value: number) => {
+    editingAlert.value.minInterval = value * 1000
+  },
+})
+
+const editingAlertDurationMs = computed({
+  get: () => editingAlert.value.minDuration,
+  set: (value: number) => {
+    editingAlert.value.minDuration = value
+  },
+})
+
+const isDurationEnabled = computed({
+  get: () => editingAlert.value.minDuration > 0,
+  set: (enabled: boolean) => {
+    editingAlert.value.minDuration = enabled ? 1000 : 0
+  },
+})
+
+const isFormValid = computed(() => {
+  return (
+    editingAlert.value.name.trim() !== '' &&
+    editingAlert.value.conditions.length > 0 &&
+    editingAlert.value.conditions.every(
+      (c) => c.variableId !== '' && c.operator !== undefined && c.compareValue !== undefined
+    )
+  )
+})
+
+const alertLevelOptions = [
+  { title: 'Success', value: AlertLevel.Success },
+  { title: 'Info', value: AlertLevel.Info },
+  { title: 'Warning', value: AlertLevel.Warning },
+  { title: 'Error', value: AlertLevel.Error },
+  { title: 'Critical', value: AlertLevel.Critical },
+]
```

Comment:
Is there not a way to do this programatically for all the alert levels?

It seems problematic to spell them out here - otherwise we need to maintain this if the levels change at all, which seems to somewhat defeat the point of having an object that defines them.